src/System.Net.Security/tests/FunctionalTests/CertificateChainValidation.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Sockets;
+using System.Net.Test.Common;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Security.Tests
+{
+    public static class CertificateChainValidation
+    {
+        public static void Validate(X509Certificate2Collection expected, X509Chain actual)
+        {
+            Debug.WriteLine("Validating certificate chain.");
```

Comment:
It probably doesn't matter as much since it's test code and not product code.  But is the `Debug.WriteLine` temporary? Or is there a plan to add some way to turn it off and/or switch to another kind of logging, i.e. System.Net test ETW logging (that I thought we had already).